sys/tools/syscalls/bsdio.lua

```diff
@@ -0,0 +1,144 @@
+--
+-- SPDX-License-Identifier: BSD-2-Clause
+--
+-- Copyright (c) 2024 Tyler Baxter <agge@FreeBSD.org>
+--
+--
+-- Thanks to Kyle Evans for his makesyscall.lua in FreeBSD which served as
+-- inspiration for this, and as a source of code at times.
+--
+-- SPDX-License-Identifier: BSD-2-Clause-FreeBSD
+--
+-- Copyright (c) 2019 Kyle Evans <kevans@FreeBSD.org>
+--
+
+util = require("util")
+
+local bsdio = {}
+
+bsdio.__index = bsdio
+
+-- Wrapper for lua write() best practice. For a simpler write call.
+function bsdio:write(line)
+	assert(self.bsdio:write(line))
+end
+
+--
+-- A write macro for the PAD64 preprocessor directive. 
+-- PARAM: bool, TRUE to pad
+-- USAGE: Pass config.abiChanges("pair_64bit") and padding will be done if 
+-- necessary. Useful for 32-bit configurations.
+--
+function bsdio:pad64(bool)
+    if bool then
+	    self:write(string.format([[
+#if !defined(PAD64_REQUIRED) && !defined(__amd64__)
+#define PAD64_REQUIRED
+#endif
+]]))
+    end
+end
+
+
+-- Returns the generated tag. Useful if only the tag is needed.
+function bsdio:tag()
+    return self.tag
+end
+
+bsdio.storage_levels = {}
+
+-- Store lines for later writing. Can also specify which order (level) to store
+-- in.
+-- Useful when a file has multiple stages of generation.
+function bsdio:store(str, level)
+    level = level or 1 -- default to level one if not provided
```

Comment:
Let's just consolidate the comment on this line up with that above the function. Maybe:

```
Can also specify which order (level) to store in, which defaults to one if not provided.
```

Reply:
There are other comments that could use a similar suggestion. I've changed those to match this suggestion (and this one). Please inform me if you see anything else.